### DISCUSSION TITLE
SnackbarStack PushAsync Example

### DISCUSSION THREAD

**Discussion by bastigerner** [Megabit/Blazorise#1605] [Q&A] [answered]
Hey,

I like the new idea of pusing a snackbar to a stack async!
Still I need some help... can someone send me a exmaple for the PushAsync call? Especially for the `Action<SnackbarStackOptions>` options parameter.

Thanks

**Comment by stsrki** [comment, marked as answer]
It's pretty simple.

```cshtml
<Button Color="Color.Success" Clicked="@(()=>snackbarStack.PushAsync("Some success message!", "Hello", SnackbarColor.Success, (options) => { options.CloseButtonIcon = IconName.Times; options.ActionButtonText = "OK"; }))">
  Show Success message
</Button>
```

**Reply by bastigerner** [reply]
Thanks thought so but was kinda to dumb to figure it out....
